src/zope/testrunner/tests/testrunner-ex/sampletests_buffering.py

```diff
@@ -30,7 +30,7 @@ def tearDown(self):
 class Layer2:
 
     def setUp(self):
-        pass
+        time.sleep(0.1)
```

Comment:
:(

Also, this appears to be [insufficient for PyPy3](https://travis-ci.org/zopefoundation/zope.testrunner/jobs/63174193), maybe they optimize `time.sleep()` too well ;)